pkg/minikube/bootstrapper/bsutil/kverify/kverify.go

```diff
@@ -37,16 +37,18 @@ const (
 	NodeReadyKey = "node_ready"
 	// KubeletKey is the name used in the flags for waiting for the kubelet status to be ready
 	KubeletKey = "kubelet"
+	// OperationalKey is the name used for waiting for pods in CorePodsList to be Ready
+	OperationalKey = "operational"
```

Comment:
how about we name this, something like
"extra"

Reply:
sounds good - renamed